src/Symfony/Component/ErrorCatcher/ErrorRenderer/ErrorRendererInterface.php

```diff
@@ -14,23 +14,19 @@
 use Symfony\Component\ErrorCatcher\Exception\FlattenException;
 
 /**
- * Interface implemented by all error renderers.
+ * Interface for classes that can render errors in a specific format.
  *
  * @author Yonel Ceruto <yonelceruto@gmail.com>
  */
 interface ErrorRendererInterface
```

Comment:
I think it's confusing that `ErrorRenderer` does not implement `ErrorRendererInterface`. The naming is bad. Maybe renaming ``ErrorRendererInterface` to `FormatSpecificErrorRendererInterface` would help but that sounds very verbose.

Reply:
`ErrorFormatterInterface`?

Reply:
`FormatErrorRendererInterface`?

Reply:
we should be the `Renderer` suffix, it's the main action.

Reply:
```php
class JsonErrorRenderer implements FormatErrorRendererInterface
```
it looks good to me :)

Reply:
Good idea. But I would propose to keep `ErrorRendererInterface` to be inline with `FragmentRendererInterface` and instead rename `ErrorRenderer` to `ErrorFormatter`.

Reply:
I'd suggest `ErrorHandler` to be inline with `FragmentHandler` :) but that isn't good because we already have a `ErrorHandler` class.

`ErrorFormatter` it also suggests to change the `render()` method to `format()`, :-1:

Reply:
I don't think we need to change `render()` to `format()` because alot of classes use `render` to transform something to string, e.g. Twig.
`FormatErrorRendererInterface` would still have ErrorRenderer in it which is confusing with the ErrorRenderer class.
I think the way I did it now is the clearest and doesn't require renaming all properties etc.